app/views/templates/_form.html.erb

```diff
@@ -84,6 +83,74 @@
       <% end %>
     </div>
 
+    <div class="tab-pane" id="help">
+      <div>
+        <%= _('Provisioning templates are generated using Embedded Ruby (ERB) language.') %>
+        <%= _('The following table and example snippet shows global methods, variables and class methods and how to use them.') %>
+        <%= _('These are guaranteed to work via the safe mode rendering, to ensure a template can do nothing harmful.') %>
+        <% unless Setting[:safemode_render] -%>
+          <%= _('To enable safe mode, navigate to Settings page and flip Safemode rendering option.') %>
+        <% end -%>
+        <code class="transparent"><pre>
+<%%# this is a comment %>
+
+<%%# use Ruby for code flow %>
+<%% if true -%>
+  some content
+<%% end %>
+
+<%%# print data from Ruby %>
+<%%= "some content from variable" %>
+
+<%%# host variable is available in most templates (Host::Managed table entry) %>
+<%%= @host %>
+
+<%%# associated operating system major version (Operatingsystem table entry) %>
+<%%= @host.operatingsystem.major %>
+
+<%%# associated hostgroup name (Hostgroup table entry) %>
```

Comment:
"hostgroup" is inconsistent with the next description, so use "host group" in the first bit of the text, but leave "Hostgroup" in parentheses as that's the class name in the table